addons/binding/org.openhab.binding.tadoac/src/main/java/org/openhab/binding/tadoac/handler/TadoACHandler.java

```diff
@@ -197,20 +198,21 @@ private synchronized void refreshSetting() {
                 }
                 updateState(CHANNEL_MODE, new DecimalType(mode));
             }
-            if (newSetting.getTemperature() != null) {
-                setting.setTemperature(newSetting.getTemperature());
-                updateState(CHANNEL_TEMP, new DecimalType(newSetting.getTemperature().getCelsius()));
+            TadoACTemperature newTemp = newSetting.getTemperature();
+            if (newTemp != null) {
+                setting.setTemperature(newTemp);
+                updateState(CHANNEL_TEMP, new DecimalType(newTemp.getCelsius()));
             }
+            updateStatus(ThingStatus.ONLINE);
         } catch (Exception e) {
-            logger.error("error while polling for new state", e);
+            logger.info("error while polling for new state", e);
```

Comment:
This is clearly an error case, but I think we do not want to flood the log files (`refreshSetting` method is called very frequently), so `debug` severity` would be sufficient, unless you have the distinction of error vs bad credentials as mentioned before.